ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -223,11 +223,143 @@ def deploy_workload(self):
         config.switch_acm_ctx()
         run_cmd(f"oc create -k {self.workload_subscription_dir}")
         run_cmd(f"oc create -k {self.workload_subscription_dir}/{self.workload_name}")
-        if self.is_placement:
-            self.add_annotation_to_placement()
-            run_cmd(f"oc create -f {self.drcp_data_yaml.name}")
+
         self.verify_workload_deployment()
 
+    def deploy_workloads_on_managed_clusters(
+        self, primary_cluster=True, secondary_cluster=False
+    ):
+        """
+        Deployment specific to busybox workload on both primary and secondary clusters
+
+        Args:
+            primary_cluster(bool) : True if apps needs to be deployed on primary cluster
+            secondary_cluster(bool) : True if apps needs to be deployed on secondary cluster
+
+        """
+        self._deploy_prereqs()
+        workload_namespaces = []
+
+        # By default, it deploys apps on primary cluster if not set to false
+        clusters = [self.preferred_primary_cluster]
+        if secondary_cluster:
+            clusters = [self.preferred_secondary_cluster]
+        if primary_cluster and secondary_cluster:
+            clusters.append(self.preferred_secondary_cluster)
```

Comment:
This might be little bit more effective and readable:
```suggestion
        clusters = [self.preferred_primary_cluster] if primary_cluster else []
        if secondary_cluster:
            clusters.append(self.preferred_secondary_cluster)
```

Reply:
Also not sure, if we shouldn't check and raise some exception, when both `primary_cluster` and `secondary_cluster` will be set to `False`.